extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -209,6 +209,30 @@ public void build(CombinedIndexBuildItem indexBuildItem, BuildProducer<Reflectiv
             //ignore, Apicurio Avro is not in the classpath
         }
 
+        try {
+            Class.forName("io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler", false,
+                    Thread.currentThread().getContextClassLoader());
+
+            reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, true,
+                    "io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler",
+                    "org.apache.kafka.common.security.oauthbearer.internals.OAuthBearerRefreshingLogin",
+                    "org.apache.kafka.common.security.oauthbearer.OAuthBearerToken"));
+
+            reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, true,
+                    "org.keycloak.jose.jws.JWSHeader",
```

Comment:
Does that mean that Strimizi Auth depends on keycloak? How can you be sure that these classes are loadable?

Reply:
Yeah,  Strimzi's `kafka-oauth-client` uses keycloak lib and adds it as a dependency. So it should be available if the library is used.